ansible/roles/haproxy/templates/haproxy.cfg.j2

```diff
@@ -3,7 +3,7 @@
 ####################
 
 frontend front-api-proxy
-    bind 0.0.0.0:8443
+    bind {{ keepalived_vip }}:8443
```

Comment:
Is it worth adding the VIP as an additional bind and leaving `0.0.0.0:8443` as an explicit bind?

Reply:
I'd argue that you only want your haproxy to listen on a specific IP address (the VIP) in this case. If your server has multiple IPs in various VLANs, you would not want to expose the kube api within VLANs where it should not be exposed to.

Perhaps if there's a use-case for it, we could make it configurable where you could provide a list of bind ip/port combinations.